service/workspace/src/workspaceservice.cpp

```diff
@@ -23,9 +23,14 @@ void WorkspaceServiceHandler::getWorkspaces(std::vector<WorkspaceInfo>& _return)
   {
     std::string filename = it->path().filename().native();
 
+    if (!fs::is_directory(filename))
```

Comment:
If I know correctly a directory will be a workspace if it contains a `project_info.json` file. Maybe we can skip directory which does not contain this file too.

Reply:
Good idea, and yes, that's the expected format. (Fun side note: Sqlite databases which are created with relative paths such as `--database "sqlite:database=helloworld.sqlite"` are put into this directory too, and cause all the fuss on the webserver if the user selects the **sqlite file** as their project.